src/components/Application-Header/JobsAppBar.tsx

```diff
@@ -1,30 +1,94 @@
 import * as React from 'react';
-
+import { withStyles, WithStyles, createStyles } from '@material-ui/core';
 import AppBar from '@material-ui/core/AppBar';
 import Toolbar from '@material-ui/core/Toolbar';
 import Typography from '@material-ui/core/Typography';
-
+import AccessibilityNewIcon from '@material-ui/icons/AccessibilityNew';
+import Avatar from '@material-ui/core/Avatar';
+import IconButton from '@material-ui/core/IconButton';
+import StarIcon from '@material-ui/icons/Star';
+import Tooltip from '@material-ui/core/Tooltip';
 import { User } from '../../types/userTypes';
 
-interface AppBarData {
+const styles = () => createStyles({
```

Comment:
Since file is a bit long... I would split it to JobsAppBar.tsx and JobsAppBarStyles.ts 😄 

This way, it will be shorter!

Reply:
Done in  b2dae38abe8cdeb64cc1b3a8fc7f1c469cfdad29